src/terrain/generation/noise.cpp

```diff
@@ -38,21 +42,28 @@ double
 generation::NoiseGenerator::smoothed_noise_(
     size_t i, NoiseTileIndex x, NoiseTileIndex y
 ) const {
-    double corners = (get_double(i, x - 1, y - 1) + get_double(i, x + 1, y - 1)
-                      + get_double(i, x - 1, y + 1) + get_double(i, x + 1, y + 1))
-                     / 16,
-           sides = (get_double(i, x - 1, y) + get_double(i, x + 1, y)
-                    + get_double(i, x, y - 1) + get_double(i, x, y + 1))
-                   / 8,
-           center = get_double(i, x, y) / 4;
+    // clang-format off
+    double corners = (get_double(i, x - 1, y - 1)
+                    + get_double(i, x + 1, y - 1)
+                    + get_double(i, x - 1, y + 1)
+                    + get_double(i, x + 1, y + 1))
+                        / 16;
+    double sides = (get_double(i, x - 1, y)
+                  + get_double(i, x + 1, y)
+                  + get_double(i, x, y - 1)
+                  + get_double(i, x, y + 1))
+                      / 8;
+    // clang-format on
+    double center = get_double(i, x, y) / 4;
     return corners + sides + center;
 }
 
 double
 generation::NoiseGenerator::interpolate_(
     NoisePosition a, NoisePosition b, NoisePosition x
 ) const { // cosine interpolation
-    double ft = x * 3.1415927, f = (1 - cos(ft)) * 0.5;
+    double ft = x * 3.1415927;
```

Comment:
```suggestion
    double ft = x * std::numbers::pi;
```

https://en.cppreference.com/w/cpp/numeric/constants